torch/csrc/jit/script/compiler.h

```diff
@@ -134,6 +149,7 @@ TORCH_API void defineMethodsInModule(
 // same as above but parse the definitions from source
 TORCH_API void defineMethodsInModule(Module & m, const std::string& source, const Resolver& resolver, std::shared_ptr<SugaredValue> self);
 TORCH_API std::shared_ptr<Graph> compileFunction(Def def, const Resolver& resolver);
+TORCH_API std::shared_ptr<Graph> compileFunction(const std::string& source);
```

Comment:
Just like in python this API should only ever return a jit::script::Module with methods on it. Graphs and their execution are an implementation detail we don't want in the public API. 

For end users the API should probably be something like:
```
std::shared_ptr<jit::script::Module> compile(std::string str) {
   std::shared_ptr<jit::script::Module> foo = new jit::script::Module();
   defineMethodsInModule(...);
   return foo;
}
```